### DISCUSSION TITLE
# FetchError: invalid json response body at http://localhost:3000/api/catalog reason: Unexpected token I in JSON at position 0FetchError: invalid json response body at http://localhost:3000/api/catalog reason: Unexpected token I in JSON at position 0

### DISCUSSION THREAD

**Discussion by salmova** [vercel/next.js#14421] [Help]
# Bug report

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

## System information

- OS: [e.g. macOS, Windows]
- Browser (if applies) [e.g. chrome, safari]
- Version of Next.js: [e.g. 6.0.2]
- Version of Node.js: [e.g. 10.10.0]

**Comment by jamesmosier** [comment]
Hi! Do you have a repository or some code you could provide? The error is that `/api/catalog` is returning a bad response, but without seeing the code it is impossible to know for sure.
